ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/PropertiesList.java

```diff
@@ -189,7 +190,8 @@
       STATE_DIR_CONFIG,
       TASK_TIMEOUT_MS_CONFIG,
       WINDOW_SIZE_MS_CONFIG,
-      UPGRADE_FROM_CONFIG
+      UPGRADE_FROM_CONFIG,
+      KSQL_NEW_QUERY_PLANNER_ENABLED
```

Comment:
Was looking into other feature flag PRs, that also add it here, but not sure why?

Reply:
Is this for LD?

Reply:
I think this is related to the `show properties` statement.This class is used in `ListPropertiesExecutor`, so all properties listed here are displayed by that statement. 

Should this flag be hidden from clients? If so, then we don't need to add it in this list.

Reply:
These are properties that are alterable after starting ksql

Reply:
After some follow up discussion, we should not add the config here, as we don't want users to change it via `ALTER SYSTEM` -- we also added it to `ImmutableProperties` to disallow changing it via `SET` and thus should not "allow list" is here.

Will fix in a follow up PR.